Description: Correctly detect mipsel architecture.
 This was spotted by the following test failure:
 .
 N7CppUnit10TestCallerI14SHA1EngineTestEE.testSHA1
 "DigestEngine::digestToHex(engine.digest()) ==
  "a9993e364706816aba3e25717850c26c9cd0d89d""
 in "src/SHA1EngineTest.cpp", line 6
Origin: vendor
Forwarded: no
Author: Cristian Greco <cristian@debian.org>

--- a/Foundation/include/Poco/Platform.h
+++ b/Foundation/include/Poco/Platform.h
@@ -165,7 +165,11 @@
 	#define POCO_ARCH_LITTLE_ENDIAN 1
 #elif defined(__mips__) || defined(__mips) || defined(__MIPS__) || defined(_M_MRX000)
 	#define POCO_ARCH POCO_ARCH_MIPS
-	#define POCO_ARCH_BIG_ENDIAN 1
+	#if defined(__MIPSEB__)
+		#define POCO_ARCH_BIG_ENDIAN 1
+	#else
+		#define POCO_ARCH_LITTLE_ENDIAN 1
+	#endif
 #elif defined(__hppa) || defined(__hppa__)
 	#define POCO_ARCH POCO_ARCH_HPPA
 	#define POCO_ARCH_BIG_ENDIAN 1
